.github/workflows/release.yml

```diff
@@ -20,20 +25,25 @@ jobs:
         run: ./mvnw ${MAVEN_ARGS} versions:set -DnewVersion="${RELEASE_VERSION:1}" versions:commit
         env:
           RELEASE_VERSION: ${{ github.event.release.tag_name }}
-      - name: Release Maven package
-        uses: samuelmeuli/action-maven-publish@v1
-        with:
-          maven_profiles: "release"
-          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
-          gpg_passphrase: ${{ secrets.GPG_PASSPHRASE }}
-          nexus_username: ${{ secrets.OSSRH_USERNAME }}
-          nexus_password: ${{ secrets.OSSRH_TOKEN }}
+  #      - name: Release Maven package
+  #        uses: samuelmeuli/action-maven-publish@v1
+  #        with:
+  #          maven_profiles: "release"
+  #          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+  #          gpg_passphrase: ${{ secrets.GPG_PASSPHRASE }}
+  #          nexus_username: ${{ secrets.OSSRH_USERNAME }}
+  #          nexus_password: ${{ secrets.OSSRH_TOKEN }}
 
   # This is separate job because there were issues with git after release step, was not able to commit changes. See history.
   update-working-version:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        if: ${{ startsWith(github.event.release.tag_name, 'v1.' ) }}
+        with:
+          ref: "v1"
+      - uses: actions/checkout@v2
+        if: ${{ startsWith(github.event.release.tag_name, 'v2.') }}
```

Comment:
Do we really need that part? I mean v2 is now master so we should only have a special for v1 now.

Reply:
good question, this forces v2 but don't have strong opinion in this. Can remove it.

Reply:
Ahh you mean the if part? Yes that is needed so its no double checkout for v1